src/pages/mypage/Mypage.js

```diff
@@ -0,0 +1,71 @@
+import { Link } from 'react-router-dom';
+import Footer from '../../components/footer/Footer';
+import Nav from '../../components/nav/Nav';
+import Top from '../../components/top/Top';
+import MypageNotLogin from './MypageNotLogin';
+import MypageLogin from './MypageLogin';
+import './Mypage.scss';
+
+function Mypage() {
+  let yourID = sessionStorage.getItem('ID');
+
+  return (
+    <div>
+      <Top />
+      <Nav />
+      <div className="Mypage">
+        {yourID ? <MypageLogin /> : <MypageNotLogin />}
```

Comment:
```suggestion
        {yourID ? <회원이름이 나온 페이지> : !yourID ? <MypageLogin /> : <MypageNotLogin />}
```
이런 식으로 구성하면 될 것 같은데 한번 시도해보세요 ! ㅎㅎ

Reply:
구조 파악이 제대로 안되서 이상하긴한데 삼항연산자를 중첩할 수 도 있으니까 활용하시면 될 거같긴해요

Reply:
아직 시간 조금 있으니까....도전!! ㅎㅎ